xml/libvirt_connect.xml

```diff
@@ -1013,8 +1013,8 @@ dynamic_ownership = 1</screen>
     <procedure>
      <step>
       <para>
-       Create the server certificate and export it together with the CA
-       certificate as described in <xref linkend="app-kvm-certificates"/>.
+       Create the server certificate and export it<!-- together with the CA
+       certificate as described in <xref linkend="app-kvm-certificates"/>-->.
       </para>
```

Comment:
The CA certificate is needed:
Create a server certificate and export it it together with the respective CA certificate.

Reply:
Ok, corrected.